src/main/java/org/minions/devfund/Library.java

```diff
@@ -1,16 +0,0 @@
-package org.minions.devfund;
-
-/**
- * This Java source file was auto generated by running 'gradle buildInit --type java-library'.
- */
-public class Library {
```

Comment:
Why are you deleting this code ?
Only commit your changes.

Reply:
done